<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RazunaDialog</class>
 <widget class="QWidget" name="RazunaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Razuna Api Key</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Shell/Shell.qrc">
    <normaloff>:/Graphics/Images/CasparCG.ico</normaloff>:/Graphics/Images/CasparCG.ico</iconset>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelApi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Api Key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>84</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>84</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>26</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditApi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>64</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Shell/Shell.qrc"/>
 </resources>
 <connections/>
</ui>
